valmlord/gulpify
# feat: add production and dev builds

## Description
This pull request enhances the Gulp setup by introducing separate build tasks for both production and development environments. It ensures that the project can be optimized for production deployment while maintaining a convenient development workflow.

## Changes
- Created new Gulp tasks:
  - `build:prod`: Optimizes assets (e.g., minification, concatenation) for production.
  - `build:dev`: Generates unminified assets with source maps for development.
- Updated the `gulpfile.js` to include conditional logic based on the build environment.

## How to Test
1. Execute the relevant Gulp tasks (e.g., `gulp build:prod`, `gulp build:dev`) to generate production or development builds.
2. Verify that the output files match the expected behavior for each environment.

## Additional Notes
- This update aims to improve the project's build process by providing tailored builds for different deployment scenarios.
